tests/unit/CompanyProvisions.spec.ts

```diff
@@ -31,7 +35,6 @@ describe('company provisions', () => {
     const wrapper = shallowMount(CompanyProvisions,
       {
         vuetify,
-        store,
```

Comment:
Since we no longer need to inject store into the component instance, does that mean the store is "external"?

Reply:
Yes, we don't use the vuex store at all - sbc-header uses it though and imports it in sbc-common-components

Reply:
We don't need the Vuex store, because none of them use them anymore. They all use the Pinia store which is loaded in via `setActivePinia(createPinia())`